frontend/src/components/pages/GlobalCodeReviewDashboard/FilterByNamesFilters.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
```

Comment:
This component has a bit of a funny name. How about just "FilterArea" or something like that? We might want to filter by other things as time goes by.  Eg we might want to filter by timestamp or review counts or something like that later on. All the filters should be together.